
<launch>
  <!-- MoveIt move_group --> 
  <include file="$(find bender_moveit_config)/launch/demo.launch"/>
  <!-- Capability Map file -->
  <param name="capmap_file" type="string" value="$(find bender_arm_planning)/database/cap_map.dat" />
  <!-- Capa de servicios de planning -->

  <!-- r_arm -->
  <node name="r_arm_planning" pkg="bender_arm_control" type="bender_planning" output="screen" args="--planning_group r_arm">
    <rosparam command="load" file="$(find bender_arm_control)/config/grasp_data.yaml"/>
  </node>

  <!-- l_arm -->
  <node name="l_arm_planning" pkg="bender_arm_control" type="bender_planning" output="screen" args="--planning_group l_arm">
    <rosparam command="load" file="$(find bender_arm_control)/config/grasp_data.yaml"/>
  </node>

  <node name="grasp_test" pkg="bender_arm_control" type="test_marker_grasp.py" output="screen">
    <param name="frame_id" value="bender/base_link"/>
  </node>

  <!--
  <node name="capmap" respawn="true" pkg="bender_arm_planning" type="capmap_service" output="screen" />
  -->

</launch>